cmd/agent.go

```diff
@@ -27,9 +31,37 @@ var agentInfoCmd = &cobra.Command{
 	},
 }
 
+var agentRBACCmd = &cobra.Command{
+	Use:   "rbac",
+	Short: "print RBAC",
+	Long:  `Print RBAC string by reading GVRs`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		file, err := os.Open(agent.ConfigFilePath)
+		if err != nil {
+			log.Fatalf("Failed to load config file for agent from: %s", agent.ConfigFilePath)
+		}
+		defer file.Close()
+
+		b, err := ioutil.ReadAll(file)
+		if err != nil {
+			log.Fatalf("Failed to read config file: %s", err)
+		}
```

Comment:
```suggestion
		b, err := ioutil.ReadFile(agent.ConfigFilePath)
		if err != nil {
			log.Fatalf("Failed to load config file for agent from: %s", agent.ConfigFilePath)
		}
```

This could be simplified by using the [ioutil.ReadFile](https://pkg.go.dev/io/ioutil#ReadFile) function. Just skips having to open the file and defer its closure.